Multisig/UI/App/UIKit/Safe Management/Add Safe/EnterSafeAddressViewController/EnterSafeAddressViewController.swift

```diff
@@ -20,20 +20,28 @@ class EnterSafeAddressViewController: UIViewController {
     @IBOutlet private weak var actionStackView: UIStackView!
     @IBOutlet private weak var actionLabel: UILabel!
     @IBOutlet private weak var openWebsiteButton: UIButton!
+    @IBOutlet private weak var externalLinkIcon: UIImageView!
+    @IBOutlet private weak var suggestionStackView: UIStackView!
 
     private var loadSafeTask: URLSessionTask?
     private var nextButton: UIBarButtonItem!
 
     override func viewDidLoad() {
         super.viewDidLoad()
         navigationItem.title = "Load Safe Multisig"
+
```

Comment:
Why to we have those empty lines ?

Reply:
because without those empty lines I could not read the code, I added them for vertical separation